src/components/Table/cells/CapacityCell/capacityCell.scss

```diff
@@ -43,4 +43,10 @@
       }
     }
   }
+
+  & .label-4 {
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+  }
```

Comment:
SpanTooltip has these styles by itself (`spanTooltip.scss`), so probably no need in this

Reply:
updated